Sources/TensorFlow/Random.swift

```diff
@@ -18,6 +18,10 @@ import Darwin
 import Glibc
 #endif
 
+public func randomSeed() -> (Int32, Int32) {
```

Comment:
The reason I didn't add such a helper earlier was because default argument initializers cannot be `internal`, and I'm not sure a global function called `randomSeed()` is the best API. But for now, the benefit of switch switching to this seems greater than API uncertainties.

Reply:
I also had some concerns about the name, but in general I feel we should have a shared mechanism for default seeds.

Reply:
How about `randomSeedForTensorFlow()`?

Reply:
Done! :)